src/main/java/com/tabgok/heimdallr/Heimdallr.java

```diff
@@ -0,0 +1,22 @@
+package com.tabgok.heimdallr;
+
+import com.tabgok.entity.factory.MachineFactory;
```

Comment:
These already exist somewhere, you will find no non-existant packages and (hopefully) no unused imports